docs/sphinx/releases/whats_new_1_5_0.rst

```diff
@@ -107,20 +127,34 @@ contains official |cmake| targets, performance improvements, and many bug fixes.
 * A new example is demonstrating how to disable thread stealing during the
   execution of parallel algorithms
   (``quickstart/disable_thread_stealing_executor.cpp``)
+* We now require for our |cmake| build system configuration files to be
+  formatted using cmake-format.
+* We have removed more dependencies on various Boost libraries. If compiled
+  in C++17 mode (``-DCMAKE_CXX_STANDARD=17``), HPX now does not require linking
+  with any Boost libraries anymore. The library still depends on a number of
+  header-only Boost libraries.
```

Comment:
This part about Boost is not new and not accurate. It also still requires a new enough Boost version and a new enough compiler (support for `std::filesystem`).